salt/modules/junos.py

```diff
@@ -912,6 +968,167 @@ def install_config(path=None, **kwargs):
     return ret
 
 
+def check_config(path=None, **kwargs):
```

Comment:
Cant we just club these features in install_config itself. As we have in ansible module.